src/wavm/syscalls.cpp

```diff
@@ -176,16 +176,12 @@ I32 executeSyscall(int syscallNumber,
             return s__rename(a, b);
         case 39:
             return s__mkdir(a, b);
-        case 45:
```

Comment:
This whole `syscalls.cpp` file is the old way we used to do syscalls. It can be removed once the dust has settled on this current work.